### ISSUE TITLE
Infrastructure: Donor experience flow testing

### ISSUE THREAD

**Issue by stephaniemcv** [MozillaFoundation/Advocacy#414]
Detailed description of work: Review donor experience flow based on different audiences and develop testing plan.
Give - Optin- Share

_Start date: December 6, 2016_ 
_Due date: December 8, 2016_ 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC: @vojtechmzl @valianttry

**Comment by stephaniemcv** [comment]
@roseregina 
I've reached out to accounting re: a credit card we can use, rather than our personal ones, since we're testing a live form and #622 can't assist us.
https://docs.google.com/spreadsheets/d/1GiI66l48Jt5MuQXhT2XG053qbiSndsoGf9Y4al4s-GI/edit#gid=0

**Comment by roseregina** [comment]
thanks! I can start in on the other parts of QA-ing the form tomorrow if it works to do that with the #622 cards

**Comment by Matt-Holland** [comment]
@stephaniemcv  and @roseregina can you say a sentence or two about where we are with this testing -- if it's blocked, if you've already done it, if we should plan to begin after Hawaii, etc?

**Comment by roseregina** [comment]
It is behind and really needs to happen. I'm going to continue working on it as I can while also trying not to get snowed under by responding to donate@ and askjoin@ for the first round of EOY emails. 

I think we are maybe waiting on live cards to use (from accounting) by @stephaniemcv would have to say. 

most pressing is making sure that currency passing to Stripe accurately and maybe that sawmill spits out the correct language.

**Comment by stephaniemcv** [comment]
@roseregina re-forwarded you info from accounting.

**Comment by Matt-Holland** [comment]
@roseregina please confirm that you have the credit card info you need, and give me an idea when you can do the testing.
- If you don't have the card info yet, I'll take care of that right away
- If you don't have time to do the testing, let's get on the phone with @HPaulJohnson  and figure out who can be re-tasked to help you.

**Comment by Matt-Holland** [comment]
I've started testing, using my personal credit card and PayPal account, and recording the results here:
https://docs.google.com/spreadsheets/d/1MnUrlyqCxu4J-ZyHrdcaqCQL9Jv9sCMJOQ7uwa1gGyo/edit#gid=1038589325
Reducing the priority on this ticket, and will close it if I do all the test versions.

**Comment by Matt-Holland** [comment]
I'll continue doing the testing, as @roseregina does not have available time. Closing.
